webapp-django/crashstats/productlib.py

```diff
@@ -30,6 +30,10 @@ class Product:
     # entries
     featured_versions: List[int]
 
+    # The list of [link name, link url] bug links for creating new bugs from the report
+    # view of a crash report
+    bug_links: List[List[str]]
```

Comment:
This field probably isn't validated very well. We should double-check the validation in the test.